data-es6.js

```diff
@@ -12960,7 +12960,7 @@ exports.tests = [
         typescript: true,
         traceur: true,
         edge15: false,
-        chrome50: true,
```

Comment:
can we add `chrome48: false` as well, assuming that's the case?

Reply:
I think it should be `chrome47: "flagged"`.

Reply:
Perfect